yarn.lock

```diff
@@ -36,7 +36,7 @@
   dependencies:
     "@babel/highlight" "^7.0.0"
 
-"@babel/core@7.4.0":
+"@babel/core@7.4.0", "@babel/core@>=7.1.0", "@babel/core@^7.1.0", "@babel/core@^7.2.2":
```

Comment:
I haven't tested if this breaks prod builds yet, but any `package.json` or `yarn.lock` changes need to be carefully wetted. So unless this is absolutely required for the functionality of the new color picker, please reset `yarn.lock` to the version on current master.